test/test.js

```diff
@@ -86,6 +86,19 @@ test('passes headers', (t) => {
   })
 })
 
+test('request has rawHeaders', (t) => {
+  t.plan(2)
+  const dispatch = function (req, res) {
+    res.writeHead(200, { 'Content-Type': 'text/plain' })
+    res.end(req.rawHeaders.toString())
```

Comment:
Instead of returning a value, could you add an assertion here that verifies that `rawHeaders` is an array?

Reply:
done